code/style.css

```diff
@@ -1,4 +1,414 @@
 html {
-  background: #0025ff;
+  background: rgba(63, 83, 110, 0.932);
+  color: rgb(255, 255, 255);
+}
+
+body {
+  margin: 0px;
+  padding: 0px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
```

Comment:
Would be super nice if the content of both containers were aligned:

<img width="303" alt="Skärmavbild 2021-01-29 kl  16 18 40" src="https://user-images.githubusercontent.com/51529201/106292940-b85c6980-624d-11eb-80e2-12ae957bdc84.png">